src/docs/components/review-stars.hbs

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  {{> html-head title="Review Stars" }}
+</head>
+
+<body class="mobile-wrapper">
+  <div class="container mt-4">
+    <h1>Review Stars</h1>
+    <span class="review-stars review-stars--primary">
+      <i class="sni sni__star"></i>
+      <i class="sni sni__star"></i>
+      <i class="sni sni__star"></i>
+      <i class="sni sni__star"></i>
+      <i class="sni sni__star"></i>
```

Comment:
we have many instances of stars across the website and we have 5 DOM elements only for that, it's too much. we need to find a solution where the svg icon is used as background and repeated 5 times or create an icon with 5 stars already